packages/push-notifications/README.md

```diff
@@ -0,0 +1,71 @@
+<p align="center">
+  <br />
+  <img
+    alt="Hyperledger Aries logo"
+    src="https://raw.githubusercontent.com/hyperledger/aries-framework-javascript/aa31131825e3331dc93694bc58414d955dcb1129/images/aries-logo.png"
+    height="250px"
+  />
+</p>
+<h1 align="center"><b>Aries Framework JavaScript Push Notifications Plugin</b></h1>
+<p align="center">
+  <a
+    href="https://raw.githubusercontent.com/hyperledger/aries-framework-javascript-ext/main/LICENSE"
+    ><img
+      alt="License"
+      src="https://img.shields.io/badge/License-Apache%202.0-blue.svg"
+  /></a>
+  <a href="https://www.typescriptlang.org/"
+    ><img
+      alt="typescript"
+      src="https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg"
+  /></a>
+    <a href="https://www.npmjs.com/package/@aries-framework/push-notifications"
+    ><img
+      alt="@aries-framework/push-notifications version"
+      src="https://img.shield.io/npm/v/@aries-framework/push-notifications"
+  /></a>
+
+</p>
+<br />
+
+Push Notifications plugin for [Aries framework JavaScript](https://github.com/hyperledger/aries-framework-javascript.git).
+
+### Installation
+
+Make sure you have set up the correct version of Aries Framework JavaScript according to the AFJ repository. To find out which version of AFJ you need to have installed you can run the following command. This will list the required peer dependency for `@aries-framework/core`.
+
+```sh
+npm info "@aries-framework/push-notifications" peerDependencies
+```
+
+Then add the push-notifications plugin to your project.
+
+```sh
+yarn add @aries-framework/push-notifications
+```
+
+### Quick start
+
+In order for this plugin to work we have to inject it into the agent to access agent functionality. See the example for more information
+
+### Example of usage
+
+> This is the current way however this will be changed someday to improve plugin management
+
+```ts
+// Agent already has been setup BUT NOT YET INITIALIZED
+import { PushNotificationsModule } from '@aries-framework/push-notifications'
```

Comment:
I'd add an agent = new Agent({ /** agent config .. */ }) line to make it more clear you create the agent before calling resolve